.travis.yml

```diff
@@ -154,11 +164,6 @@ jobs:
       workspaces:
         use: dockerconfig
 
-    - <<: *create_manifest
-      name: Create manifest for master snapshot for Quay
-      if: (branch = master) AND (tag IS blank)
-      env: IMAGE=$OAO_IMAGE_QUAY TAG="snapshot"
-
```

Comment:
Aren't we supposed to leave Quay and delete the DockerHub's manifest?

Reply:
Quay is now the default (see line 157) ;)

Reply:
Ah, missed that.